modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/eventTypes.js

```diff
@@ -12,10 +12,15 @@
  * details.
  */
 
-const SIDEBAR = {
+export const SIDEBAR = {
```

Comment:
It is not necessary to export `SIDEBAR`, because we are already adding within the scope `EVENT_TYPES`, so we must call `EVENT_TYPES.SIDEBAR...`.

In DXP, this can cause problems, as it can be used by other modules and starts to create a usage contract that we don't want, so it is good to keep things inside just the file or module.